charts/ti-key-release-1/templates/configmap/tsi-mutate-configmap.yaml

```diff
@@ -24,7 +24,7 @@ data:
           value: {{ .Values.secrets.refreshSec }}
         volumeMounts:
           - name: tsi-secrets
-            mountPath: /usr/share/secrets
+            mountPath: /usr/share/secrets/tsi-secrets
```

Comment:
does the consumer of this have to be changed somewhere?

Reply:
Since we always enforce the local path for secrets in the container must start with `tsi-secrets`, this name would always be appended to the path in the sidecar, so it would be always properly processed by  `components/jwt-sidecar/get-vault-secrets.sh` script on the sidecar, then mounted to the container.